style.css

```diff
@@ -0,0 +1,12 @@
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css");
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
```

Comment:
Instead try to stick to one specific version and only use those icons?

Reply:
Sure Hamza, I'll refactor it later on... Thanks for suggestion

Reply:
Sure Hamza, I'll refactor it later on... Thanks for suggestion